packages/web-app-pdf-viewer/src/App.vue

```diff
@@ -1,80 +1,28 @@
 <template>
-  <main>
-    <app-top-bar v-if="!loading && !loadingError" :resource="resource" @close="closeApp" />
-    <loading-screen v-if="loading" />
-    <error-screen v-else-if="loadingError" />
-    <div v-else class="oc-height-1-1">
-      <object class="pdf-viewer oc-height-1-1 oc-width-1-1" :data="url" type="application/pdf" />
-    </div>
-  </main>
+  <AppWrapper
+    id="pdf-viewer"
```

Comment:
Can we move this into the AppWrapper component and default it to application-id?

Reply:
I'll mark this as resolved by...you^^